docs/navigate/devops-testing/toc.yml

```diff
@@ -431,6 +431,8 @@ items:
             href: ../../core/deploying/native-aot/diagnostics.md
           - name: Native code interop
             href: ../../core/deploying/native-aot/interop.md
+          - name: Building native libraries
```

Comment:
```suggestion
          - name: Build native libraries
```

Reply:
> We're not supposed to using -ing words in titles.

It's the title of the page this links to. Should that one also be changed?

Reply:
Ideally, yes. But it's not the end of the world.